web-sdk.js

```diff
@@ -1839,17 +1839,12 @@
         }
 
         function r() {
-            var e, t, n, i, o = m.document.cookie,
-                r = {};
-            if (!o || "string" != typeof o) return {};
-            o = o.split(/;\s/);
-            for (e = o.length; e--;) try {
-                t = o[e].match(/^([^=]+)(=(.*))?$/);
-                if (!t) continue;
-                n = p(t[1]);
-                i = p(t[3] || "");
-                r[n] = i;
-            } catch (e) { }
+            var n, i, r = {};
+            for (var key in localStorage) try {
```

Comment:
This iterates all enumerable properties in the `localStorage` object, including `getItem`, `setItem`, etc.

To iterate over just the set of key/vals that were stored, the only way that is spec compliant is verbose:
```javascript
for (let i = 0; i < localStorage.length; i++) {
  const key = localStorage.key(i);
  const val = localStorage.getItem(key);
  // ...
}
```